### ISSUE TITLE
Identify what is slowing down solidproject.org updates

### ISSUE THREAD

**Issue by RubenVerborgh** [solid/process#226]
#217 filed a concern that updates to solidproject.org are not happening fast enough.

Part of the problem is addressed by simply following the process. For instance, https://github.com/solid/process/pull/195 makes clear that typos do not require review.

In order fix other issues, we need to understand what they are first. A solution proposed in #222 suggests that they would be related to 1) editor reaction speed and 2) lack of constructive feedback, but as I have stated there, I don't think those are the right problems (and hence #222 cannot be the right solution).

So before jumping to solutions, let's first make sure we have the right problems.

**Comment by RubenVerborgh** [comment]
In order to inform this process, I ran some quick statistics. I have collected the time to first review for every PR on solidproject.org using a [script](https://gist.github.com/RubenVerborgh/55835d36a5fb25a0f34cbfbb5934d910), and published the [current raw results](https://gist.github.com/RubenVerborgh/c60714d4fd59a5cb5dc8e383540c5a66).

These numbers allow us to objectively observe that: 
- The median review time so far has been 1 hour and 21 minutes.
- 50 out of 111 PRs (45%) were reviewed in less than 1 hour.
- 60 out of 111 PRs (54%) were reviewed in less than 2 hours.
- 78 out of 111 PRs (70%) were reviewed in less than 12 hours.
- 85 out of 111 PRs (76%) were reviewed in less than 24 hours.
- In June and July 2020, 31 out of 35 PRs (89%) were reviewed in less than 24 hours.
  - Two of those were reviewed within 2 hours of assignment (https://github.com/solid/solidproject.org/pull/216#pullrequestreview-433182935 and https://github.com/solid/solidproject.org/pull/233#pullrequestreview-437201526).
  - One did not require editor review according to the process (https://github.com/solid/solidproject.org/pull/283#issuecomment-662383155).
   - One was missing details from the contributor and took 4 days (https://github.com/solid/solidproject.org/pull/271#pullrequestreview-450593728).
  - As such, I round this up to 31 out of 32 (97%) timely actions taken.

**TL;DR: The numbers show that reviews are being delivered _on time_.**
Therefore, let's keep looking for actual causes that do need fixing.

**Comment by RubenVerborgh** [comment]
I also created a similar [script](https://gist.github.com/RubenVerborgh/77610660a674f98767f0f9f2406a252b) to extract review comments from all PRs, and published the [current raw results](https://gist.github.com/RubenVerborgh/8a56f4f4d56284a9c9c492dfd1d1a3cb) in order to identify how often non-constructive comments appear.

There are 171 comments in total for 111 PRs, excluding explicit approvals (since those do not request changes).

Analyzing whether a comment is constructive is, however, a subjective matter, so I cannot provide objective numbers here. Therefore, I invite others to count.

Nonetheless, here are some objective measurements:
- the word "suggestion" appears 53 times
- the word "fix(ed/ing)" appears 15 times
- 40 question marks appear (indicative of questions / dialog)
These numbers seem positive to me.

On a subjective note, I did not directly identify non-constructive comments myself.

**TL;DR: Found objective evidence of constructive comments; no non-constructive comments (subjective) identified.**

**Comment by Mitzi-Laszlo** [comment]
@RubenVerborgh what is the time frame used to generate the data you mentioned? 

What are the results when you split it between 2019 vs. 2020 considering there was a process change at the turn of the year?

**Comment by RubenVerborgh** [comment]
> @RubenVerborgh what is the time frame used to generate the data you mentioned?

As you can see from the raw data, which I've linked: all.

> What are the results when you split it between 2019 vs. 2020 considering there was a process change at the turn of the year?

It's open data, so feel free to make any analyses you want.

That said, a simple "sorry, I indeed was wrong suggesting editors are slow" also works for me.
